dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandlerTest.kt

```diff
@@ -84,4 +86,57 @@ internal class LogcatLogHandlerTest {
                     "\$runnable\$1"
             )
     }
+
+    @Test
+    fun `resolves valid stack trace element when wrapped in timber`(forge: Forge) {
+
+        // Given
+        val forgeFileName: Forge.() -> String = {
+            "${this.anAlphabeticalString(Case.ANY)}.${this.aStringMatching("(kt|java)")}"
```

Comment:
you should escape the "."

Reply:
why? it is not a regex pattern

Reply:
hmm...maybe I don't interpret it correctly but if you want to have a "." in there you should add it as `\\.` otherwise it will be interpret as the special character `.` which means any character.

Reply:
this is just a plain string literal and not regex pattern, so no need to escape it here. I'm just using lambda with `Forge` receiver.